components/TextResponse.tsx

```diff
@@ -28,12 +28,23 @@ const TextResponse = ({
 
   return (
     <>
-      {title && <h3>{title}</h3>}
+      {title && (
+        <h3>
+          <label htmlFor={title}>{title}</label>
+        </h3>
+      )}
       <textarea
         className={styles.textbox}
+        id={title}
         name={title}
         onChange={(e) => updateUserText(e.target.value)}
         placeholder={placeholder}
+        // Focus the textarea on render
```

Comment:
That didn't happen when clicking next from the previous question (both on Chrome and Firefox).

Reply:
It works for me on both Chrome and Firefox, although not with hot reload (had to clear cache and reload)